config/prow/plugins.yaml

```diff
@@ -74,6 +74,15 @@ label:
   - proposal/0.17
   - proposal/0.18
   - proposal/0.19
+  - channel/kafka
```

Comment:
It looks Prow does not support setting additional_labels for a repo... Can you add a comment here to say these labels are specific to eventing?

Reply:
done